hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/main/java/org/apache/hadoop/yarn/server/nodemanager/recovery/NMLeveldbStateStoreService.java

```diff
@@ -1813,4 +1813,22 @@ protected void checkVersion() throws IOException {
             + getCurrentVersion() + ", but loading version " + loadedVersion);
     }
   }
+  @Override
+  public void releaseAssignedResources(ContainerId containerId, String resourceType)  {
+    LOG.info("Release assigned resources for a container of a specified resourceType");
```

Comment:
This has to be debug to avoid verbose logs.

    if (LOG.isDebugEnabled()) {
          LOG.debug(
          "releaseAssignedResources: containerId=" + container.getContainerId() + " resourceType=" + resourceType);
    }